src/utils/url.ts

```diff
@@ -24,3 +24,11 @@ export const venueRoomUrl = (venue: WithId<AnyVenue>, roomTitle: string) => {
   );
   return venueRoom ? venueRoom.url : venueInsideUrl(venue.id);
 };
+
+export const isExternalUrl = (url: string) =>
+  url.includes("http") &&
+  new URL(window.location.href).host !== new URL(getRoomUrl(url)).host;
+
+const getRoomUrl = (roomUrl: string) => {
+  return roomUrl.includes("http") ? roomUrl : "//" + roomUrl;
```

Comment:
Not sure.. but will this logic cover all any potential edge cases we may hit? Is there a better way to 'construct' the URL?

Reply:
I'm not sure as well, this was already written and I just moved it outside of the component to make it reusable, because I've seen it in quite a lot of places.